DOCS-2854: Setting up DKIM email authentication (#1938)
MultiSafepay/docs: content/my-account/manage-account/adding-spf-records.md

---
title : "Adding SPF records for MultiSafepay emails"
weight: 13
meta_title: "Adding SPF records - MultiSafepay Docs"
read_more: "."
url: '/my-account/adding-spf-records/'
aliases:
    - /faq/general/add-spf-dns-records
    - /faq/general/adding-spf-dns-records/
    - /developer/general/adding-spf-dns-records/
---

Sender Policy Framework (SPF) records let you specify who is authorized to send emails on your domain's behalf. Receiving email servers can check the SPF record to verify the sender. Using an SPF record prevents emails sent by MultiSafepay on your behalf from being marked as spam.

## Adding an SPF record

> 1. Through your hosting provider, domain registrar, or DNS provider, create a DNS TXT record that is named after your domain. For example: `example.com`

Reviewer: ```suggestion
1. Through your hosting provider, domain registrar, or DNS provider, create a DNS TXT record named after your domain, e.g. `example.com`.
```
